tagged-localbuild.xml

```diff
@@ -733,30 +725,36 @@
+  <project name="vendor-qcom-opensource-telephony" path="vendor/qcom/opensource/telephony" remote="sony" revision="06062e65442a05f1469ed59085e2217045061c03" upstream="aosp/LA.UM.9.12.r1" dest-branch="aosp/LA.UM.9.12.r1" groups="device,local::qcom"/>
+  <project name="vendor-qcom-opensource-time-services" path="vendor/qcom/opensource/time-services" remote="sony" revision="9002b6d7b1fc1cb775f7302cfe66e5bdf472b86b" upstream="aosp/LA.UM.9.12.r1" dest-branch="aosp/LA.UM.9.12.r1" groups="device,local::qcom"/>
+  <project name="vendor-qcom-opensource-usb" path="vendor/qcom/opensource/usb" remote="sony" revision="3ecd1faf8f4fcef9cd61e534342b155365d7c86d" upstream="aosp/LA.UM.9.12.r1" dest-branch="aosp/LA.UM.9.12.r1" groups="device,local::qcom"/>
+  <project name="vendor-qcom-opensource-vibrator" path="vendor/qcom/opensource/vibrator" remote="sony" revision="b6ac417de16816e14cc9169a63903a9ec7ceef7d" upstream="aosp/LA.UM.7.1.r1" dest-branch="aosp/LA.UM.7.1.r1" groups="device,local::qcom"/>
+  <project name="vendor-qcom-opensource-wlan-fw-api" path="kernel/sony/msm-4.14/kernel/drivers/staging/wlan-qc/fw-api" remote="sony" revision="2b3b2e23608d2f392cf50a0d2958be21965b080d" upstream="aosp/LA.UM.8.1.r1" dest-branch="aosp/LA.UM.8.1.r1" groups="device,local::LA.UM.7.1.r1"/>
+  <project name="vendor-qcom-opensource-wlan-fw-api" path="kernel/sony/msm-4.19/kernel/drivers/staging/wlan-qc/fw-api" remote="sony" revision="7b58b37877560bbb3916a1e73624c045b51ad21a" upstream="aosp/LA.UM.9.12.r1" dest-branch="aosp/LA.UM.9.12.r1" groups="device,local::LA.UM.9.12.r1"/>
+  <project name="vendor-qcom-opensource-wlan-qca-wifi-host-cmn" path="kernel/sony/msm-4.14/kernel/drivers/staging/wlan-qc/qca-wifi-host-cmn" remote="sony" revision="bdbcbc6512e482968e6ca9427333ee72481bd90a" upstream="aosp/LA.UM.8.1.r1" dest-branch="aosp/LA.UM.8.1.r1" groups="device,local::LA.UM.7.1.r1"/>
+  <project name="vendor-qcom-opensource-wlan-qca-wifi-host-cmn" path="kernel/sony/msm-4.19/kernel/drivers/staging/wlan-qc/qca-wifi-host-cmn" remote="sony" revision="31be74b57ea702a3c35ca466c08808c8a5366a1b" upstream="aosp/LA.UM.9.12.r1" dest-branch="aosp/LA.UM.9.12.r1" groups="device,local::LA.UM.9.12.r1"/>
+  <project name="vendor-qcom-opensource-wlan-qcacld-3.0" path="kernel/sony/msm-4.14/kernel/drivers/staging/wlan-qc/qcacld-3.0" remote="sony" revision="fcc0c7c1349e9573520fa34d08f18a992df65170" upstream="aosp/LA.UM.8.1.r1" dest-branch="aosp/LA.UM.8.1.r1" groups="device,local::LA.UM.7.1.r1"/>
+  <project name="vendor-qcom-opensource-wlan-qcacld-3.0" path="kernel/sony/msm-4.19/kernel/drivers/staging/wlan-qc/qcacld-3.0" remote="sony" revision="7902f2dad725f8d8a5c49b8db673bbf8edd1f57b" upstream="aosp/LA.UM.9.12.r1" dest-branch="aosp/LA.UM.9.12.r1" groups="device,local::LA.UM.9.12.r1"/>
```

Comment:
These submodules will cause conflicts with kernel submodules from droid-hal repo.

Reply:
Oh those slipped in.